lib/standardFs.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const fs = require('fs');
+const mkdirp = require('mkdirp');
```

Comment:
Shouldn’t this be added as a dependency?

Reply:
Done